danger-klaxit/lib/danger_plugin.rb

```diff
@@ -19,6 +19,7 @@ def common
     warn_for_public_methods_without_specs
     warn_for_bad_order_in_config
     warn_rubocop
+    warn_for_not_updated_structure_sql
```

Comment:
this looks quite important to me. What do you think?
```suggestion
    fail_for_not_updated_structure_sql
```